### ISSUE TITLE
Grid outside of layer boundaries.

### ISSUE THREAD

**Issue by webdobe** [danzel/Leaflet.utfgrid#21]
I have a base map layer that fits the whole map div (which has no data caked in).  I have added a layer that in this case is just 1 state surrounded by nothing but the base map.  When you hover outside of the state map an empty data object is being passed and throwing errors for every mouse movement.  The problem happens at:

line 145ish of source.
var idx = this._utfDecode(data.grid[gridY].charCodeAt(gridX)),

basically the object passed for "data" is empty I believe the code at 142 needs to be looking for a null data object and an empty.
